examples/js/trx/consolidate-account-balances.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Tron wallet can't send funds from their receive addresses.
+ * Account balance consolidation is used to sweep funds from the
+ * receive addresses into the wallet's base address for sending.
+ * @see {@link https://app.bitgo.com/docs/#operation/v2.wallet.consolidateaccount.build}
+ */
+const BitGoJS = require('bitgo');
+const Promise = require('bluebird');
+
+// TODO: change to 'production' for mainnet
+const bitgo = new BitGoJS.BitGo({ env: 'test' });
+
+// TODO: change to 'trx' for mainnet
+const coin = 'ttrx';
+
+// TODO: set your wallet id
+const walletId = '';
+
+// TODO: set your wallet passphrase
+const walletPassphrase = '';
+
+// TODO: set OTP code
+const otp = '000000';
+
+// TODO: set your access token here
+// You can get this from User Settings > Developer Options > Add Access Token
+const accessToken = '';
+
+Promise.coroutine(function *() {
```

Comment:
Using Coroutines / blue bird is not required, we can use simple async / awaits.

Reply:
Ok, updated.